sal_interfaces/Watcher/Watcher_Events.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+    <Enumeration>AlarmSeverity_None,AlarmSeverity_Warning,AlarmSeverity_Serious,AlarmSeverity_Critical</Enumeration>

```

Comment:
So we had only two types of severity in the requirements doc: LTS-897-REQ-0009 - Critical & Warning

Reply:
I think we need 3 levels and @tribeiro agrees, so I one of us will request a change of the requirements.

Reply:
We can leave for now.   Please enter a CR today.  Thanks.